test/registry.js

```diff
@@ -1,5 +1,21 @@
-const RegistryFetcher = require('../lib/registry.js')
 const t = require('tap')
+// Stub out sigstore verification for testing to avoid needing to refresh the tuf cache
+const RegistryFetcher = t.mock('../lib/registry.js', {
+  sigstore: {
+    sigstore: {
+      verify: async (bundle) => {
```

Comment:
I looked at the coverage report to see why we weren't at 100% and found that the only thing not being exercised was the invocation of the `keySelector` callback (which makes sense since we've totally stubbed out the `sigstore.verify` implementation).

<img width="678" alt="image" src="https://user-images.githubusercontent.com/398027/217679832-a600a1ab-a140-4e8f-933f-101aca13d796.png">

Not sure if we want to do this, but I found I could get things back to 100% simply by making sure we invoke the passed-in callback:
```suggestion
      verify: async (bundle, data, options) => {
        options.keySelector && options.keySelector()
```

Reply:
Great spot! Seems like we might need to do something like this while we are stubbing this out entirely 👍